client/src/components/carousel.jsx

```diff
@@ -6,7 +6,6 @@ class Carousel extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-
```

Comment:
@aku2nnya Understanding this is a placeholder for a future feature that requires state, I may consider removing it until you need it!

Reply:
The style guide detects it as needing it, so I'll have it in for now (since I think I'll be making use of it soon anyways), but thanks for noticing.